packages/internet-header/cypress/support/prepare-story.ts

```diff
@@ -15,14 +15,13 @@ export const prepare = (
 ) => {
   installInterceptors(config);
   cy.visitStorybook({
-    onBeforeLoad(win: { navigator: any }) {
-      // Set default browser language explicitly to English
-      Object.defineProperty(win.navigator, 'language', {
-        value: 'en',
-      });
```

Comment:
This never works actually because it's the latest priority from the list

Reply:
This should work if nothing else is specified.

Reply:
Yes, but in this case, we always get the `document.documentElement.lang` and it has a higher priority.

Reply:
Only if it is specified on the `<html>` element. I added html documents to the fixtures where it is not specified on purpose (https://github.com/swisspost/design-system/blob/main/packages/internet-header/cypress/fixtures/pages/no-language-set.html). `document.documentElement.lang` is an empty string in those cases.